components/common/VerifyAndMintSection.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Dispatch, SetStateAction, useEffect } from 'react';
+import { NftForm } from '../../utils/Interfaces';
+import Button from '../global/Button';
+
+interface VerifyAndMintSectionProps {
+  formData: NftForm;
+  setRenderConfirmMint: Dispatch<SetStateAction<boolean>>;
+}
+
+const VerifyAndMintSection = function ({
+  formData,
+  setRenderConfirmMint,
+}: VerifyAndMintSectionProps) {
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
+  return (
+    <div>
+      <h1 className="font-bold text-3xl">Verify and Mint</h1>
+      <h3 className="my-8">Your NFT</h3>
+      <picture>
+        <source src={URL.createObjectURL(formData.nftFiles[0])} />
+        <img
+          src={URL.createObjectURL(formData.nftFiles[0])}
+          alt="overlay"
+          className="rounded-md w-full h-1/2 mb-8"
+        />
+      </picture>
+      <section className="mb-16">
+        <p className="text-xs text-gray-500">Creator: {formData.creatorName}</p>
+        <p className="text-xl font-semibold">{formData.tokenName}</p>
+        <p>{formData.displayName}</p>
+      </section>
+      <p className="text-sm ">
+        You are minting {formData.nftFiles.length}
```

Comment:
The amount of NFTs minted is not the number of files assigned to your NFT. One NFT can have multiple files assigned. Please change this so we count the # of NFTs on the collection minting flow and simply say 1 on the single NFT flow.

Reply:
Done